packages/es-components/rollup.config.js

```diff
@@ -35,33 +35,35 @@ export default [
       'react-modal',
       'tinycolor2',
       'get-root-node-polyfill/implement',
-      'format-message'
+      'format-message',
     ],
     plugins: [
       wildcardExternal([
         '@babel/**',
         'lodash/**',
         'core-js/**',
-        'text-mask-addons/**'
+        'text-mask-addons/**',
       ]),
       babel({
         exclude: ['node_modules/**'],
-        babelHelpers: 'runtime'
-      })
-    ]
+        babelHelpers: 'runtime',
+      }),
+    ],
   },
   {
     input: 'src/index.js',
     output: {
       file: 'bundle/main.min.js',
+      banner:
+        'var self = this.styled ? this : self; var globalThis = self || globalThis;',
```

Comment:
This had to generate `self` and `globalThis` in order to work with the new version of es-components _and_ old versions of isolated-externals.